sk_sidekick/lib/src/commands/release_command.dart

```diff
@@ -108,12 +108,21 @@ ${changelog.readAsStringSync().replaceFirst('# Changelog', '').trimLeft()}''');
         .start(workingDirectory: repository.root.path);
     final newChangelogAndVersionBranch = _getCurrentBranch(repository.root);
 
-    // TODO locking + tagging on separate branch <package name>-release
+    // locking + tagging on separate branch <package name>-release
     final releaseBranch = '${package.name}-release';
-    // TODO correct command is: git checkout $releaseBranch || git checkout -b $releaseBranch
-    // TODO but '...'.start treats the first word ("git") as command and passes the rests as args to it; this isn't what I want to do though -> ||
-
-    print(' - Tagging release ($tag)...');
+    final tag = '${package.name}-v$nextVersion';
+    print(' - Update release branch $releaseBranch and tag ($tag)...');
+    // This is a workaround because '<cmd1> <args1> || <cmd2> <args2>'.run
+    // actually executes cmd1 with the arguments [<args1>, <cmd2>, ||, <args2>]
+    final shell = env['SHELL'];
+    if (shell == null) {
+      throw "Couldn't determine which shell to run";
+    }
+    '$shell git checkout $releaseBranch || git checkout -b $releaseBranch'.run;
```

Comment:
Explicitly using `sh` should work, too. 

Alternatively, split it in two commands and checking the exit code of the first